src/blocks/content-helper/content-helper-provider.ts

```diff
@@ -138,22 +140,25 @@ class ContentHelperProvider {
 		if ( tag ) {
 			return ( {
 				query: { limit, tag },
-				message: `${ __( 'with the tag', 'wp-parsely' ) } "${ tag.name }"`,
+				/* translators: %s: message such as "with tag Foo" */
+				message: sprintf( __( 'with tag "%1$s"', 'wp-parsely' ), tag.name ),
 			} );
 		}
 
 		// A category exists.
 		if ( category?.name ) {
 			return ( {
 				query: { limit, section: category.name },
-				message: `${ __( 'in the category', 'wp-parsely' ) } "${ category.name }"`,
+				/* translators: %s: message such as "from category Foo" */
+				message: sprintf( __( 'from category "%1$s"', 'wp-parsely' ), category.name ),
```

Comment:
Since posts could be in multiple categories, would this be better as:

```suggestion
				message: sprintf( __( 'in category "%1$s"', 'wp-parsely' ), category.name ),
```

Reply:
I think this

```
Top-performing posts from category "Developers" in last 3 days.
```

is better than this

```
Top-performing posts in category "Developers" in last 3 days.
```

but please do confirm if you think change should be done.

Reply:
I don't know which is better.

Reply:
"In" sounds better to me and the sample survey of my 11yo twins!

Reply:
Feedback Implemented.